packages/f/ffmpeg/scripts/gas-preprocessor.pl

```diff
@@ -0,0 +1,1243 @@
+#!/usr/bin/env perl
+# by David Conrad
```

Comment:
it's too large, please use xmake's script.

Reply:
But it's only available on xmake dev, isn't that an issue?

Reply:
Before xmake is released, you can download it as a resource first

Reply:
I'm not on my computer for now, I will change it asap